app/validation.py

```diff
@@ -149,10 +155,26 @@ def get_validation_errors(validator_name, json_data,
 
 
 def min_price_less_than_max_price(error_map, json_data):
-    if 'priceMin' in json_data and json_data.get('priceMax'):
-        if 'priceMin' not in error_map and 'priceMax' not in error_map:
-            if Decimal(json_data['priceMin']) > Decimal(json_data['priceMax']):
-                return {'priceMax': 'max_less_than_min'}
+
+    def return_min_and_max_price_keys(json_data):
+        if 'priceMin' in json_data:
+            return 'priceMin', 'priceMax'
+
+        specialist_price_keys = [k for k in json_data.keys() if k.endswith(('PriceMin', 'PriceMax'))]
```

Comment:
Neat, I didn't know 'endswith' accepts tuples. I think we could use this for the `not_money_format` check too.